Dockerfile

```diff
@@ -49,7 +49,7 @@ RUN mkdir -p ${HOME} && \
 USER devel:devel
 WORKDIR $APP
 
-RUN gem install bundler -v 2.1.4
+RUN gem install bundler
```

Comment:
We don't need to change this but there is an option I like to use that is `gem install bundler -v $(tail -n 1 Gemfile.lock)`